### DISCUSSION TITLE
Strange behavior with ^SPX history

### DISCUSSION THREAD

**Discussion by ghost** [dpguthrie/yahooquery#185] [General]
Hello,
The feature history has a very strange behavior with ticker '^SPX'
When you want a daily history, it returns only the current price
`tickers = yf.Ticker('^SPX')`
`dfMTM= tickers.history(period = '1mo', interval = '1d')`
but when you set the interval lower than daily, you get the whole period
`df= tickers.history(period = '1mo', interval = '1h')`

**Comment by dpguthrie** [comment]
Appears to be a YF issue and not anything related to the package.  You can always check behavior on YF to see if you're seeing something similar.

https://www.loom.com/share/32f697aa6c61441aa44390e97a340e3d
